setup.cfg

```diff
@@ -139,6 +135,12 @@ all =
     paramiko
     snowflake-sqlalchemy>=1.4.4  # Temporary solution for https://github.com/astronomer/astronomer-providers/issues/958, we should pin apache-airflow-providers-snowflake version after it pins this package to great than or equal to 1.4.4.
 
+docs =
+    sphinx
+    sphinx-autoapi
+    sphinx-copybutton
+    astronomer-providers[all]
```

Comment:
Hmm, this is recursive, no?

You just need to install:

```
pip install -e .[docs]
```

Reply:
no, it does not cause a recursion as `all` does not include the dependencies for `docs`, but the `docs` now need all the packages to be installed to evaluate the attributes on the classes.